src/pages/components/dropdown/usage.mdx

```diff
@@ -25,18 +31,27 @@ import items from "../../../data/components/dropdown.js";
 
 ## Overview
 
-Dropdowns present a list of options that can be used to filter or sort existing content.
+There are three different types of dropdowns that support various kinds of functionality—dropdown, multiselect, and combobox.
```

Comment:
are we supposed to use a different type of dash here? seems strange to read "...functionality-dropdown..."

Reply:
Yes, this Jan did a content review on that.